theano/scalar/basic.py

```diff
@@ -3843,7 +3843,7 @@ def __init__(self, inputs, outputs):
         self.nin = len(inputs)
         self.nout = len(outputs)
         self.init_fgraph()       # self.fgraph
-
+        self.init_c_code()
```

Comment:
I think you removed the wrong line. If that isn't the case, this is very strange, so commit the code you tested.